### ISSUE TITLE
Clean documentation in docs/

### ISSUE THREAD

**Issue by ilyamandel** [TeamCOMPAS/COMPAS#686]
There is some documentation in docs (rather than docs/online-docs) that is outdated and no longer needed. For example, https://github.com/TeamCOMPAS/COMPAS/blob/dev/docs/getting_started.md is out of date (see, e.g., reference to non-existing .pdf file in the final paragraph).  Rather than updating it, and always remembering that we have two versions to update, we should check that it's not referenced elsewhere and delete it.

**Comment by ilyamandel** [comment]
@jeffriley , @reinhold-willcox -- 
It seems that all of the information in https://github.com/TeamCOMPAS/COMPAS/blob/dev/docs/getting_started.md is indeed repeated in the new readthedocs documentation at https://compas.science/docs/.  However, there are several other .md files in https://github.com/TeamCOMPAS/COMPAS/tree/dev/docs which contain information not available at readthedocs as far as I can tell.  I think we should have documentation in one place, though, and they are too hidden here.  Can you please check whether any of this information should be transferred to readthedocs before these are deleted?

**Comment by reinhold-willcox** [comment]
Sorry, I haven't kept up with all the activity on github! I am making changes to the postProcessing folder & notebooks, so I will add on to clean up the docs/ folder as well (there are a lot of hyperlinks back and forth, so it makes sense to do them in parallel).
